runtime/gc_structs/ClassIterator.hpp

```diff
@@ -72,17 +72,19 @@ class GC_ClassIterator {
 	GC_CallSitesIterator _callSitesIterator;
 	GC_MethodTypesIterator _methodTypesIterator;
 	GC_MethodTypesIterator _varHandlesMethodTypesIterator;
+	bool _shouldScanClassObject; /**< Boolean needed for balanced GC to prevent ClassObject from being scanned twice  */
```

Comment:
I believe you can add "const" modifier here. It might improve compiled code.

Reply:
👍